files/en-us/web/html/global_attributes/anchor/index.md

```diff
@@ -83,7 +83,7 @@ body {
 }
 ```
 
-We use CSS to convert the `infobox` element into an **anchor-positioned element** and position it relative to its anchor. We set its:
+We use CSS to convert the `infobox` element into an _anchor-positioned element_ and position it relative to its anchor. We set its:
```

Comment:
since we don't define it, ** was not appropriate, but i felt it needed to stick out here.